src/components/autofill/autofill.ts

```diff
@@ -36,11 +36,13 @@ css/* scss */ `
       font-size: 12px;
       pointer-events: none;
       white-space: nowrap;
+      visibility: visible;
```

Comment:
this is not a good approach. You are setting the vibisility of the whole component  to "invisible" and at the same time, you make the children `div`'s visible. if anything, it probably questions the current structure of the component.